vector/src/main/java/im/vector/activity/VectorRoomActivity.java

```diff
@@ -854,11 +855,21 @@ public void onClick(View v) {
             }
         });
 
+        mAllowVoip = getResources().getBoolean(R.bool.allow_voip);
```

Comment:
This boolean `mAllowVoip` should be removed. It is useless. 
`mSession.isVoipCallSupported()` is more relevant.
Please revert your changes in `VectorRoomActivity.java`.

With the current implementation the call icons will be hidden for the flavor "withoutVoip".
 Indeed:
- mSession.isVoipCallSupported() returns false because native libraries are not included.
- the CallsManager checks this boolean before placing a call, so no call will be placed